modules/rostests/kmtests/fltmgr/fltmgr_create/fltmgr_create.c

```diff
@@ -0,0 +1,147 @@
+/*
+ * PROJECT:         ReactOS kernel-mode tests - Filter Manager
+ * LICENSE:         GPLv2+ - See COPYING in the top level directory
```

Comment:
I plan to change the framework's license to LGPLv2+ in the near future (and update the license headers to the new format using SPDX identifiers in the process). If you're okay with that as the license, it might make sense to make the change now.

Reply:
yep, re-license it however you want